packages/core/src/common/message-service-protocol.ts

```diff
@@ -21,12 +21,17 @@ export interface Message {
     type: MessageType;
     text: string;
     actions?: string[];
+    timeout?: number;
```

Comment:
shouldn't that be a `options?: MessageOptions` property?

Reply:
Will be name options: MessageOptions